website/content/api-docs/system/replication/replication-dr.mdx

```diff
@@ -438,13 +438,34 @@ $ curl \
 
 ## Update DR secondary's primary
 
-This endpoint changes a DR secondary cluster's assigned primary cluster using a
-secondary activation token. This does not wipe all data in the cluster.
+The update endpoint changes the primary cluster assigned to a DR
+secondary cluster. Changing the primary cluster assignment does not
+ wipe data in the secondary cluster.
 
 This endpoint requires a DR Operation Token to be provided as means of
 authorization. See the [DR Operation Token API
 docs](#generate-disaster-recovery-operation-token) for more information.
 
+There are two ways to update the primary assignment:
+
+1. **Use a secondary activation token with the `token` parameter**. An
+    activation token is required after a DR failover when updating a newly
+    demoted DR secondary cluster.
+
+1. **Use primary cluster addresses with the `update_primary_addrs` parameter**.
+    During the update, cluster addresses are pinged one at a time via gRPC.
+    The first cluster to respond successfully is assigned as the new primary
+    address. Updating with cluster addresses is not allowed after a DR
+    failover when updating a newly demoted DR secondary cluster. This was added
+    in Vault 1.13.3.
+
+The two update methods are mutually exclusive. You may use one or the
+other, but not both. In most cases, update-primary is called on a DR
+cluster after a failover, which will require the use of an activation
+token. If you're confused as to which method to use, a good rule of
+thumb is to use `token` on DR secondary clusters and `update_primary_addrs`
+on performance secondary clusters.
```

Comment:
```suggestion
other, but not both. A good rule of thumb is to use `token` on DR secondary
clusters and `update_primary_addrs` on performance secondary clusters.
```

It's not generally helpful to tell folks what they will do in most cases unless determining the right course of action is confusing. In this case, it's not needed since you're providing a good way for readers to determine the appropriate solution and we've explicitly mentioned that tokens are required in specific situations.

Style correction: write in active voice